bundles/org.eclipse.passage.loc.operator/plugin.xml

```diff
@@ -21,15 +21,7 @@
             application="org.eclipse.e4.ui.workbench.swt.E4Application">
          <property
                name="appName"
-               value="%application.name">
```

Comment:
I think this change was silently applied by PDE and needs to be reverted

Reply:
reverted